### ISSUE TITLE
Update AMDS Interface to AMDC

### ISSUE THREAD

**Issue by codecubepi** [Severson-Group/AMDS#47]
- [x] Create "`amds`" development branch off of `v1.3-staging` in AMDC-Firmware
- [x] Create branch off of `develop` in AMDS repo
- [x] Get working AMDC+AMDS setup and probe the interface between the boards to understand the timing
- [x] Update AMDS's firmware to send data back to AMDC's new IP driver over UART immediately after SPI communications from daughtercards are complete

Also see https://github.com/Severson-Group/AMDC-Firmware/issues/365

**Comment by npetersen2** [comment]
@codecubepi will make a new issue which documents the testing which needs to be completed before merging the PR linked to this.

**Comment by codecubepi** [comment]
See [#388](https://github.com/Severson-Group/AMDC-Firmware/issues/388) and [#389](https://github.com/Severson-Group/AMDC-Firmware/issues/389) in the AMDC-Firmware repo.
